tests/integration/security/authorization_test.go

```diff
@@ -862,9 +862,12 @@ func TestAuthorization_TCP(t *testing.T) {
 				// - request to port http-8090 should be denied because both path and port are matched.
 				// - request to port http-8091 should be allowed because the port is not matched.
 				// - request to port tcp-8092 should be allowed because the port is not matched.
-				newTestCase(a, b, "http-8090", false, scheme.HTTP),
-				newTestCase(a, b, "http-8091", true, scheme.HTTP),
-				newTestCase(a, b, "tcp-8092", true, scheme.TCP),
+				newTestCase(a, b, "/data", "http-8090", false, scheme.HTTP),
+				newTestCase(a, b, "/data", "http-8091", true, scheme.HTTP),
+				newTestCase(a, b, "/data", "tcp-8092", true, scheme.TCP),
+				newTestCase(a, b, "/regexp/", "http-8090", false, scheme.HTTP),
```

Comment:
no, we shouldn't use path in the TestAuthorization_TCP test, the /data before is just an placeholder and not used in the authz policy, it might work here as this specific test case is using HTTP port but that is not the purpose of this test (HTTP is covered in other tests). 

To test regex in TCP, just use those attributes that are available in TCP, e.g. source principal or namespace, and to distinguish with other tests you can create new app.